client/my-sites/controller.js

```diff
@@ -116,10 +116,11 @@ function renderEmptySites( context ) {
 	);
 }
 
-function renderNoVisibleSites( context ) {
+function renderNoVisibleSites( context, getState ) {
```

Comment:
`getState` is available off of context, so there's no need to pass this.

eg we can define below:
```jsx
const state = context.store.getState();
```